openwisp_notifications/settings.py

```diff
@@ -37,6 +37,10 @@
     'openwisp-notifications/audio/notification_bell.mp3',
 )
 
+OPENWISP_NOTIFICATIONS_EMAIL_BATCH_INTERVAL = getattr(
+    settings, 'EMAIL_BATCH_INTERVAL', 30 * 60  # 30 minutes
```

Comment:
**Thinking out loud**

I think 30 minutes is a lot of time. It will affect the real-timeness of the email notifications. Most of the consecutive notifications would be delayed by this time. We should also point this out in the README. 


@nemesifier what do you think?